### DISCUSSION TITLE
css variables to control popper animation style

### DISCUSSION THREAD

**Discussion by triptu** [radix-ui/themes#289] [Ideas]
It's really nice that we can customise different aspects of the components by overriding the [CSS Variables](https://github.com/radix-ui/themes/tree/main/packages/radix-ui-themes/src/styles/tokens) defined as part of style tokens. I was wondering if we can do this for animation open and close durations as well? Or if there is already a way to do this?

The values I want to override are hardcoded [here](https://github.com/radix-ui/themes/blob/94fdcea33dc22864c0e9e461e67085bd04e83dfb/packages/radix-ui-themes/src/components/animations.css#L96) for open and close state.

**Comment by vladmoroz** [comment]
Yeah we could look into that—just curious, were you looking to speed up or slow down the animations?

**Reply by triptu** [reply]
Speed up, there are some context and other menus which are used often enough that it would be much better for them to be near instant. Adding an example from [linear](https://linear.app/) to better convey this, note how quick the submenus are for instance - 


https://github.com/radix-ui/themes/assets/16299282/ce01cf66-414d-421a-8702-3a5d137b42ae
